test/parallel/test-http-regr-gh-2821.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const server = http.createServer(function(req, res) {
+  res.writeHead(200);
+  res.end();
+
+  server.close();
+});
+
+server.listen(common.PORT, function() {
+
+  const req = http.request({
+    method: 'POST',
+    port: common.PORT
+  });
+
+  const payload = new Buffer(16390);
```

Comment:
Can you fill it with something, please? I would prefer if random memory regions were not passed through the network stack, even in the harmless tests.